target/linux/armsr/modules.mk

```
@@ -287,6 +287,27 @@ define KernelPackage/octeontx2-net
 endef
 $(eval $(call KernelPackage,octeontx2-net))
 
+define KernelPackage/renesas-net
+SUBMENU:=$(NETWORK_DEVICES_MENU)
+TITLE:=Renesas network drivers
+DEPENDS:=@(TARGET_armsr_armv8) +kmod-phylink +kmod-mii
+KCONFIG:=NET_VENDOR_RENESAS \
+CONFIG_RAVB \
+CONFIG_PTP_1588_CLOCK=y \
+CONFIG_PTP_1588_CLOCK_OPTIONAL=y \
+CONFIG_MDIO_BITBANG=y \
+CONFIG_MDIO_GPIO \
+CONFIG_PHYLIB=y
```

```suggestion
DEPENDS:=@(TARGET_armsr_armv8) +kmod-phylink +kmod-mii +kmod-ptp +kmod-libphy +kmod-mdio-gpio
KCONFIG:=CONFIG_NET_VENDOR_RENESAS=y \
CONFIG_RAVB
```

Think you were missing `CONFIG_` from `NET_VENDOR_RENESAS`, then I made a guess at trying to clean up this package to avoid forcing tri-state modules to be built into kernel (via `=y`), and instead depending on existing (OpenWrt) packages.

PTP_1588_CLOCK: https://github.com/openwrt/openwrt/blob/9db53fb6aa261c4aa73da7d1ec6cf2d3b06b8fb8/package/kernel/linux/modules/other.mk#L1052
MDIO_GPIO: https://github.com/openwrt/openwrt/blob/9db53fb6aa261c4aa73da7d1ec6cf2d3b06b8fb8/package/kernel/linux/modules/netdevices.mk#L159
PHYLIB: https://github.com/openwrt/openwrt/blob/9db53fb6aa261c4aa73da7d1ec6cf2d3b06b8fb8/package/kernel/linux/modules/netdevices.mk#L96

---

Hi @john-tho 
Thanks for your comments.
I've updated the following:
1. I've fixed CONFIG_ from NET_VENDOR_RENESAS and removed the CONFIGs by adding the existing packages
2. CONFIG_PTP_1588_CLOCK_OPTIONAL still required because of PTP_1588_CLOCK being module